dashboard/app/models/teacher_feedback.rb

```diff
@@ -60,4 +60,22 @@ def self.latest_per_teacher
   def self.latest
     find_by(id: maximum(:id))
   end
+
+  # Increments student_visit_count and related metrics timestamps for a TeacherFeedback.
+  def increment_visit_count
+    now = DateTime.now
+
+    if student_visit_count
+      self.student_visit_count += 1
```

Comment:
why `self.` on one line but not the other? either seems ok, but seeing both makes me think there's something I'm missing

Reply:
`self.` is required when you're making an assignment, otherwise `self.` is implicit (semi-helpful [StackOverflow](https://stackoverflow.com/questions/10805136/when-to-use-self-in-model)). so lines 68 and 79 don't require `self.`, but the lines making assignments (69, 71, 75, and 78) do.

i believe we have rubocop set up to yell at you when you're unnecessarily using `self.`, which is also helpful!

Reply:
Perfect. TIL!